common/src/test/java/com/microsoft/identity/common/MicrosoftStsAccountCredentialAdapterTest.java

```diff
@@ -53,7 +53,9 @@
 import java.util.Date;
 import java.util.GregorianCalendar;
 
-import static com.microsoft.identity.common.internal.providers.microsoft.MicrosoftAccount.AUTHORITY_TYPE_MS_STS;
+import cz.msebera.android.httpclient.extras.Base64;
```

Comment:
I'm fine with using msebra over JoseJWT but not a fan of dropping our own Base64 class...which I believe was just a wrapper around an implementation of Base64 (which was JoseJWT until now I believe). Can we keep that wrapper? Or instead of using Base64 directly from msebra, can we create our own interface for Base64 and then provide an implementation of it using msebra? and then probably make a factory that returns Base64 impl? so basically in the future if we decide to something other than mseabra then we just need to change internal implementation and not have to change the imports in a lot of places.

Reply:
I was debating with myself on this, and eventually decided to remove it.

I could add back our class, but due to its flexibility, the interface would be almost 1:1 mapping with msebra. (i.e. we'll have to create a flag enum that mirrors theirs, or we'll end up creating a lot of helper functions, etc).